### DISCUSSION TITLE
Grouped bar chart - strange display if value is 0

### DISCUSSION THREAD

**Discussion by martinblampied** [apexcharts/apexcharts.js#3694] [Q&A]
Hello,

I'm seeing some strange display if any value in a grouped bar chart is set to 0. Rather getting a bar with value 0 it expands the width of non-zero value to take up both.

I can replicated this on the example shown here: https://apexcharts.com/javascript-chart-demos/bar-charts/grouped/

```
  var options = {
          series: [{
          data: [44, 0, 41, 64, 22, 43, 21]
        }, {
          data: [53, 32, 33, 52, 13, 0, 32]
        }],
```

Displays as:
![Screenshot 2023-03-09 at 16 05 27](https://user-images.githubusercontent.com/3713820/224083338-d4b95df0-1feb-454e-8426-235a68c1b02a.png)

Is this intended behaviour? How can I get it to display 0 properly?

**Comment by rafaelOsio** [comment]
same
